doc/pairs.md

```diff
@@ -207,3 +207,65 @@ tuples
     - 51
 ...
 ```
+## ``fields`` parameter
+
+With ``fields`` parameter you can choose the value of which fields you would like to get.
+Result contains fields specified by ``fields`` parameter, primary key and scan key. We need primary and scan keys in result to support pagination.
+
+**Example:**
+
+```lua
+objects = {}
+-- condition by indexed non-unique non-primary field
+for _, obj in crud.pairs('developers', {{'>=', 'age', 31}},  {use_tomap = true, fields = {'name'}}) do
+    table.insert(objects, obj)
+end
+
+objects
+---
+- - id: 6
+    name: Alexey
+    age: 31
+  - id: 4
+    name: Mikhail
+    age: 51
+...
+```
```

Comment:
Add a note to describe what happened, smth like "we got `name` field as it was specified, `age` field because space was scanned by `age` index and primary key `id`".